.env.staging

```diff
@@ -0,0 +1,6 @@
+REACT_APP_DATA_GENERAL='https://spasovski.net/general.json'
+REACT_APP_DATA_REVISIONS='https://spasovski.net/revisions.json'
+REACT_APP_DATA_PROBES='https://spasovski.net/all_probes.json'
+REACT_APP_DATA_ENVIRONMENT='https://spasovski.net/environment.json'
+REACT_APP_DATA_OTHER_FIELDS='https://spasovski.net/other_fields.json'
+REACT_APP_DATA_DATASETS='https://spasovski.net/datasets.json'
```

Comment:
Does CRA automatically use this file in staging? Also, if you want staging to perfectly mirror production, you'd want to ingest the same data in both. Maybe these endpoints would be better for a development server?

Reply:
This was something intended for testing that is totally unused. Will remove.